dipy/io/stateful_tractogram.py

```diff
@@ -138,6 +140,35 @@ def __init__(self, streamlines, reference, space,
         self._origin = origin
         logger.debug(self)
 
+    @staticmethod
```

Comment:
Any particular reason why this is a staticmethod instead of method? I see later you are calling it like this `self.is_compatible_sft` anyway. Also, if it stays a staticmethod, I think `are_compatible` would be a more accurate name for (or `is_compatible` if you make it a normal method).

Reply:
I have to call it self.is_compatible_sft since it is defined in that file. But elsewhere it is going to be:
```
from dipy.io.stateful_tractogram import StatefulTractogram
StatefulTractogram.are_compatible(sft_1, sft_2)
```
(I changed the name)